guides/06_other-tutorials/theming-guide.md

```diff
@@ -17,10 +17,15 @@ with gr.Blocks(theme=gr.themes.Soft()):
 	height="450"
 ></iframe>
 
+Gradio comes with a set of prebuilt themes which you can load from `gr.themes.*`. These are:
 
-Gradio comes with a set of prebuilt themes which you can load from `gr.themes.*`. Each of these themes set values for hundreds of CSS variables.
+* `gr.themes.Base()`
```

Comment:
Added a blurb on all the themes that ship with gradio

![image](https://user-images.githubusercontent.com/41651716/226463245-f9c07c85-cc05-4a36-9158-a178297aeaf7.png)

Reply:
Nice